tests/openvino/test_diffusion.py

```diff
@@ -586,7 +648,7 @@ def test_num_images_per_prompt(self, model_arch: str):
                         outputs = pipeline(**inputs, num_images_per_prompt=num_images_per_prompt).images
                         self.assertEqual(outputs.shape, (batch_size * num_images_per_prompt, height, width, 3))
 
-    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    @parameterized.expand(["stable-diffusion", "stable-diffusion-xl"])
```

Comment:
```suggestion
    @parameterized.expand(SUPPORTED_ARCHITECTURES)
```

Reply:
this test is not applicable for flux and sd3 as callback interface is deperecated in 0.31.0, these pipelines do not support it, so parametrization parameters correct